### PR TITLE
Update AWS/S3 config

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

File: backends/rapidpro/backend.go [hunk 1]
```diff
@@ -174,22 +174,16 @@ func (b *backend) Start() error {
 		s3config := &storage.S3Options{
 			AWSAccessKeyID:     b.config.AWSAccessKeyID,
 			AWSSecretAccessKey: b.config.AWSSecretAccessKey,
+			Region:             b.config.AWSRegion,
 			Endpoint:           b.config.S3Endpoint,
-			Region:             b.config.S3Region,
-			DisableSSL:         b.config.S3DisableSSL,
```
Comment (rowanseymour) [inline backends/rapidpro/backend.go:179]: Best I can tell this isn't needed if you're providing an endpoint as a full URL with http or https

File: backends/rapidpro/backend.go [hunk 2]
```diff
@@ -174,22 +174,16 @@ func (b *backend) Start() error {
...
-			ForcePathStyle:     b.config.S3ForcePathStyle,
 			MaxRetries:         3,
 		}
-		if b.config.AWSAccessKeyID != "" && !b.config.AWSUseCredChain {
-			s3config.AWSAccessKeyID = b.config.AWSAccessKeyID
-			s3config.AWSSecretAccessKey = b.config.AWSSecretAccessKey
```
Comment (rowanseymour) [inline backends/rapidpro/backend.go:185]: These are being set above anyway. The logic should be that if `AWSUseCredChain` is set, you get S3 storage if tho `AWSAccessKeyID` and `AWSSecretAccessKey` are blank.

File: backends/rapidpro/backend.go [hunk 3]
```diff
@@ -655,7 +649,7 @@ func (b *backend) SaveAttachment(ctx context.Context, ch courier.Channel, conten
 
 	orgID := ch.(*Channel).OrgID()
 
-	path := filepath.Join(b.config.S3AttachmentsPrefix, strconv.FormatInt(int64(orgID), 10), filename[:4], filename[4:8], filename)
+	path := filepath.Join("attachments", strconv.FormatInt(int64(orgID), 10), filename[:4], filename[4:8], filename)
```
Comment (norkans7) [inline backends/rapidpro/backend.go:652]: Why not keep that as `media`?
Are the files actually moved on that path now?

Comment (rowanseymour) [reply backends/rapidpro/backend.go:652]: This is just what is set on the default config (used by tests) that we always overwrite on prod to attachments. RP, mailroom and courier should all use the same `attachments` prefix.

**Review by norkans7 — APPROVED** [review]

**Pull request merged**